lib/rubocop/cop/lint/debugger.rb

```diff
@@ -72,8 +52,26 @@ def message(node)
           format(MSG, source: node.source)
         end
 
-        def binding_irb?(node)
-          binding_irb_call?(node)
+        def with_receiver?(node)
+          !!node.receiver
+        end
+
+        def special_rule?(node)
```

Comment:
I don't get what's the purpose of this `special_rule?` method.

Reply:
this is an edge case that I couldn't find a better fix. Maybe you can help me:

Even tho `Kernel` is a receiver and `save_and_open_page` is a debugger method together they should not be an offense.

Here is the test case for it: https://github.com/rubocop-hq/rubocop/blob/12bd06882eaca12bd33c77f31e8d976bc3ad0bed/spec/rubocop/cop/lint/debugger_spec.rb#L164-L166

Reply:
I'd just drop this test case. For such a small thing people should add a local exclusion rule. It just doesn't fit with having with our goal to have a configurable cop.

Reply:
Cool! I will drop it! It was introduced here. https://github.com/rubocop-hq/rubocop/commit/83902a6a4b957130abcabdebe1c266a12ec5d138